src/HealthCheck/DoctrineConnectionHealthCheck.php

```diff
@@ -34,27 +35,41 @@ class DoctrineConnectionHealthCheck implements HealthCheckInterface
 
     public function __construct(
         #[Autowire(service: 'doctrine.dbal.default_connection')]
-        private readonly ?Connection $connection
-    ) {
+        private readonly ?Connection $connection,
+    )
+    {
     }
 
     public function check(HealthReportInterface $report): HealthReportInterface
     {
-        if (!is_null($this->connection)) {
-            try {
-                // Get the schema manager and grab the first table to later query on
-                $sm = $this->connection->createSchemaManager();
-                $tables = $sm->listTables();
-                if (!empty($tables)) {
-                    $table = reset($tables);
-                    // Perform a light-weight query on the chosen table
-                    $query = "SELECT * FROM %s LIMIT 1";
-                    $this->connection->executeQuery(sprintf($query, $table->getName()));
-                }
-            } catch (Exception) {
-                return HealthReport::buildStatusDown('Unable to execute a query on the database.');
+        // Skip database checking of no database is configured.
+        if ($this->connection === null) {
+            return $report;
+        }
+
+        try {
+            $this->connection->connect(); // This will create the SQLite database if it does not exist
```

Comment:
No not exactly. And We could create probably much more checks if we decide to differentiate per DB platform.

It the database can't be reached, this throws the connection error as well for MySQL and Postgres. 

This could be skipped, as the createSchemaManager would do the same though. It was a bit clearer to me when the exception actually fails. Now it can fail on two ocasions, connect and createSchemaManager.

Reply:
In that case, then the very specific comment might need to be made less focused on sqlite?

Reply:
I made it slightly clearer I hop